src/tyxml/imandra_document_tyxml.ml

```diff
@@ -187,17 +187,98 @@ module Mapper = struct
       ?(title="doc") ?meta:(my_meta=[])
       ?(headers : Html_types.head_content_fun H.elt list =[])
       (self:t) (doc:D.t) : H.doc =
+    let b_style = H.link ~rel:[`Stylesheet]
+        ~href:"https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" ()
+    in
     let style0 =
-      let l = [
-        "table.framed { border: 2px solid black; }";
-        "table.framed th, table.framed td { border: 1px solid black; }";
-        "th, td { padding: 3px; }";
+      let l = ["
```

Comment:
use `{|` and `|}` instead of `"`, so you don't need to escape the strings within

Reply:
e.g. `{| body { font-family: .. "Segoe UI" .. } |}`